guide/popular-topics/builders.md

```diff
@@ -0,0 +1,123 @@
+# Builders
+
+discord.js provides a `builders` package which contains a variety of utilities you can use when writing your Discord bot.
+To install the package, run `npm install @discordjs/builders` in your terminal.
+
+## Formatters
+
+Formatters are a set of utility functions which format input strings into the given format.
+
+### Basic Markdown
+
+The Formatters provide functions to format strings into all the different Markdown styles supported by Discord.
+
+```js
+const { bold, italic, strikethrough, underscore, spoiler, quote, blockQuote } = require('@discordjs/builders');
+const string = 'Hello!';
+
+const boldString = bold(string);
+const italicString = italic(string);
+const strikethroughString = strikethrough(string);
+const underscoreString = underscore(string);
+const spoilerString = spoiler(string);
+const quoteString = quote(string);
+const blockquoteString = blockQuote(string);
+```
+
+### Links
+
+There are also two methods to format hyperlinks. `hyperlink()` will format the URL into a masked markdown link, and `hideLinkEmbed()` will wrap the URL in `<>`, preventing it from embedding.
+
+```js
+const { hyperlink, hideLinkEmbed } = require('@discordjs/builders');
+const url = 'https://discord.js.org/';
+
+const link = hyperlink(url);
+const hiddenEmbed = hideLinkEmbed(url);
+```
+
+### Codeblocks
+
+You can use `inlineCode()` and `codeBlock()` to turn a string into an inline codeblock or a regular codeblock with or without syntax highlighting.
+
+```js
+const { inlineCode, codeBlock } = require('@discordjs/builders');
+const jsString = 'const value = true;';
+
+const inline = inlineCode(jsString);
+const codeblock = codeBlock(jsString);
```

Comment:
```suggestion
const codeBlock = codeBlock(jsString);
```

Reply:
`codeblock` cannot be `codeBlock` here because that's the name of the function imported from the builders package

Reply:
🤔 so should it be left as `codeblock` or `codeBlockString` or smthn (might have to append the others with `String` too in that case for consistency)